### ISSUE TITLE
SmartCity-Accessibility #113 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#100]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 115 lines...]
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.017 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.socialnetwork.ReviewCommentTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.031 sec - in smartcity.accessibility.socialnetwork.ReviewCommentTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.037 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.BrokenAdminTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.184 sec - in smartcity.accessibility.socialnetwork.BrokenAdminTest
Running smartcity.accessibility.socialnetwork.MockAdminTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.35 sec - in smartcity.accessibility.socialnetwork.MockAdminTest
Running smartcity.accessibility.search.SearchTest
-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Dec 27, 2016
JxMaps license valid.
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 314.214 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.navigation.NavigationTest
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00007fe53d0f3000, 262144, 0) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 262144 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /var/lib/jenkins/workspace/SmartCity-Accessibility/hs_err_pid6143.log

Results :

Tests run: 28, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:56 min
[INFO] Finished at: 2016-12-27T11:03:53+00:00
[INFO] Final Memory: 27M/200M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/workspace/SmartCity-Accessibility && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec -jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefirebooter5022833160051376991.jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire2183030097424091810tmp /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire_02791304110232766996tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] 2b3e705a8da5a643b8fd11051c64a4130c692e41 - slight refactoring in gui package

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/113/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
